test/extended/router/scoped.go

```diff
@@ -136,7 +137,7 @@ var _ = g.Describe("[sig-network][Feature:Router]", func() {
 			pattern := "%s-%s.myapps.mycompany.com"
 
 			g.By("waiting for the healthz endpoint to respond")
-			healthzURI := fmt.Sprintf("http://%s:1936/healthz", routerIP)
+			healthzURI := fmt.Sprintf("http://%s/healthz", net.JoinHostPort(routerIP,"1936"))
```

Comment:
Gofmt will want a space here.  
```suggestion
			healthzURI := fmt.Sprintf("http://%s/healthz", net.JoinHostPort(routerIP, "1936"))
```